float32 percent_utilization
